examples/employee-app/src/main/java/io/helidon/service/employee/EmployeeRepositoryImplDB.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.service.employee;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeRepositoryImplDB implements EmployeeRepository {
+
+    private final Connection conn = DBConnection.getInstance().getConnection();
```

Comment:
Same comment as for `EmployeeRepositoryImpl` and `EmployeeMockList`, the `DBConnection` class does not have much value. There is at most one instance of `EmployeeRepository` since there is only once instance of `EmployeeService` in the application.

I suggest to get rid of `DBConnection` and move all the code to create the connection in the constructor of this class, pass the `Config` instance to the constructor as argument instead of using `Config.create`. The constructor of `EmployeeService` already takes a `Config` instance so you can pass it through.

Reply:
Done